packages/perseus/src/widgets/passage-ref-target.tsx

```diff
@@ -50,8 +50,10 @@ class PassageRefTarget extends React.Component<any> {
         return (
             <Renderer
                 content={this.props.content}
-                // @ts-expect-error - TS2322 - Type '{ content: any; inline: boolean; apiOptions: any; linterContext: any; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Renderer> & Pick<Readonly<Props> & Readonly<{ children?: ReactNode; }>, "children" | ... 3 more ... | "legacyPerseusLint"> & InexactPartial<...> & InexactPartial<...>'.
-                inline={true}
+                // This was here, but the prop was not defined until now! (#873)
+                // Perhaps it once existed and was removed a long time ago.
+                // Commenting out to prevent unanticipated side effects
+                // inline={true}
```

Comment:
This is weird! We had `inline` here already, but it didn't do anything.

Reply:
Was added waaaaaaaay back: https://phabricator.khanacademy.org/D12113

I think it makes sense to comment this out for now, like you did.  Maybe add a reference to this Phabricator diff in the comment just for posterity?

Reply:
And then removed here: https://phabricator.khanacademy.org/D13454#inline-89200